hybrid/HybridSampleApps/AccountEditor/build.gradle.kts

```diff
@@ -13,10 +13,10 @@ dependencies {
 android {
     namespace = "com.salesforce.samples.accounteditor"
 
-    compileSdk = 34
+    compileSdk = 35
```

Comment:
Not related to this but was causing errors in Android Studio

Reply:
Does the UI of hybrid apps get messed up by Edge to Edge with API 35?

Reply:
I haven't tried (yet). I was just trying to go past the error.